src/routes/TraderForm/Form/TransactionDetails/index.tsx

```diff
@@ -17,6 +19,16 @@ export const TransactionDetails: React.FunctionComponent<TransactionDetailsProps
   const infoPostSwap = useAppSelector(selectInfoPostSwap);
   const token = useAppSelector(selectAMMTokenFormatted);
   const submitButtonInfo = useAppSelector(selectSubmitButtonInfo);
+  const slippage = useAppSelector(selectSlippage);
+
+  let fee = '--';
+  if (infoPostSwap.status === 'success') {
```

Comment:
move this entire logic to selectors. Create a selector: selectInfoPostSwapFeeFormatted and add your logic there.

Reply:
[05fe837](https://github.com/Voltz-Protocol/voltz-ui/pull/587/commits/05fe837b4cb3abd9425e03bd5049214e3c9c7ea7)